kratos/tests/test_serializer.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function, absolute_import, division
+
+import os
+import sys
+
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as KratosUtils
+
+dependencies_are_available = KratosUtils.CheckIfApplicationsAvailable("FluidDynamicsApplication")
+if dependencies_are_available:
+    import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+    from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class TestSerializer(KratosUnittest.TestCase):
+    def _prepare_test(self):
+        # Define a model and load the parameters
+        self.pre_serialized_model = KratosMultiphysics.Model()
+        with open("auxiliar_files_for_python_unnitest/parameters_files/test_serializer.json",'r') as parameter_file:
```

Comment:
I am fairly sure that you need `GetFilePath` here:
```suggestion
        with open(GetFilePath("auxiliar_files_for_python_unnitest/parameters_files/test_serializer.json"),'r') as parameter_file:
```